### ISSUE TITLE
Classification Accuracy on shrec16 

### ISSUE THREAD

**Issue by kamran-afzal** [ranahanocka/MeshCNN#45]
Hi,
I've executed your source code without changing any settings. But, at 200 epoch, I'm getting 97.5% accuracy on shrec16  data set, but you've mentioned 98.6% accuracy in your paper. could you please tell me, is this 200th epoch accuracy or a highest value at any epoch of training model.

Thanks,

**Comment by ranahanocka** [comment]
Hi @kamran-afzal ,

I took the max accuracy after training to 200 epochs.
I think it usually happened around ~160 epochs? 
But don't think that it happened exactly at epoch 200 (probably overfitting by then).
